components/script/dom/websocket.rs

```diff
@@ -299,17 +299,7 @@ impl WebSocket {
         let chan = global.r().networking_task_source();
         let address = Trusted::new(self, chan.clone());
 
-        let new_buffer_amount = (self.buffered_amount.get() as u64) + data_byte_len;
-        if new_buffer_amount > (u32::max_value() as u64) {
-            self.buffered_amount.set(u32::max_value());
-            self.full.set(true);
-
-            let _ = self.Close(None, None);
-            return Ok(false);
-
-        }
-
-        self.buffered_amount.set(new_buffer_amount as u32);
+        self.buffered_amount.set(self.buffered_amount.get() + data_byte_len);
```

Comment:
Do we need to worry about the "+" overflowing here? (It isn't actually possible to send that much data in a reasonable amount of time, but it could be potentially be queued depending on how exactly the queue is implemented.)